### ISSUE TITLE
Add profiler-enabled wheel to release assets, despite it not working yet

### ISSUE THREAD

**Issue by tt-rkim** [tenstorrent/tt-metal#14301]
@ayerofieiev-tt has requested we upload a profiler-enabled build of the wheel, despite not testing it, for use in ttnn/pytorch

We can add explicit warnings to our release notes saying to not use the profiler wheel build unless you absolutely know what you're doing. We could put similar instructions in the INSTALLING.md

cc: @mo-tenstorrent @blozano-tt @afuller-TT @TT-billteng

**Comment by tt-rkim** [comment]
Will not do for now per @ayerofieiev-tt 
Thank you all for your patronage

**Comment by tt-rkim** [comment]
For the future

Discussed w/ @mo-tenstorrent 

We likely will have to make changes to the `tracy` callable module + post-proc scripts to:

- be searchable in both repo and wheel-installed environments
- Separate the concerns of generated artifacts (probably a `TT_METAL_HOME` thing) vs. finding the post-proc scripts as part of a dev's profiling workflow
- Ensure instructions are up to date
- Post-commit, t3k profiling, device perf should be sufficient to ensure these changes work

**Comment by blozano-tt** [comment]
I vote to close this issue.

**Comment by mo-tenstorrent** [comment]
We need a solution for sharing profiler post proc scripts with users before closing this. Do you have an alternative suggestion?

**Comment by blozano-tt** [comment]
> We need a solution for sharing profiler post proc scripts with users before closing this. Do you have an alternative suggestion?

I was thinking that customers who ingest a wheel file would never need to profile the innards of the library. I assumed it was only needed for internal users. Is this an incorrect assumption? Let’s talk more when time allows.

**Comment by mo-tenstorrent** [comment]
Yeah, let's chat. Whoever is programming with Metal/ttnn or tt-mlir will need access to this. Even if they are using already developed kernels, they will need to see the individual kernel perfs and in case they need to optimize a kernel, they will further use the profiler to determine what is slowing down the kernel. 

On host side as well, while our host has had a bunch of optimization rounds already, there is still a ton of place that cause bottlenecks both on python and in c++

**Comment by tt-rkim** [comment]
we now enable it by default!!! THANK YOUUUU @blozano-tt
